model/annotations_schema.js

```diff
@@ -7,74 +8,109 @@ sql.setDialect('postgres')
 exports.initDatabase = initDatabase
 
 // type to help link subtypes of annotations
-annoTypeName = "anno_sub_type"
+var annoTypeName = "anno_sub_type"
 
 // define tables here:
-annotations = sql.define({
-    name: 'annos',
+exports.annotations = annotations = sql.define({
+    name: 'annotations_annotation',
     columns: [
-	{name: 'user_id',       dataType: 'varchar(40)', notNull: true},
-	{name: 'u_id', dataType: 'serial', primaryKey: true},
-        {name: 'ref_source', 	dataType: 'varchar(20)', notNull: true},
- 	{name: 'reference',	dataType: 'varchar(45)', notNull: true},
-	{name: 'comment',       dataType: 'varchar(3000)'},
-	{name: 'type', dataType: annoTypeName, notNull: true}],
+	{name: 'id', dataType: 'serial', primaryKey: true},
+	{name: 'comment',	dataType: 'varchar(300)',},
+  	{name: 'heritable', dataType: 'varchar(8)'},
+	{name: 'measurement_type', dataType: 'varchar(30)', notNull: true},
+	{name: 'characterization', dataType: 'varchar(20)', notNull: true},
+	{name: 'reference_id',	dataType: 'integer', notNull: true},
```

Comment:
serial is a special kind of ID - it doesn't have to be.  the datatype is mostly for initializations and writes